docs/walkthrough/index.md

```diff
@@ -0,0 +1,255 @@
+subjects:
+  - kind: ServiceAccount
+    name: tiller
+    namespace: kube-system
+```
+
+- Create the service account and role binding specified in the above file with the `kubectl apply` command:
+
+```bash
+kubectl apply -f helm-rbac.yaml
+```
+
+- To deploy Tiller into your AKS cluster, use the `helm init` command:
+
+```bash
+helm init --service-account tiller
+```
+
+## Deploy Promitor to your cluster using Helm
+
+In addition to the helm chart, you'll need a values file with secrets & a metric declaration file (these can also be the same file for ease of use). The yaml file below will scrape one metric, queue length, from the queue created above.
+
+```yaml
+azureAuthentication:
+  appId: <guid-sp-app-id>
+  appKey: <guid-sp-generated-password>
+
+azureMetadata:
+  tenantId: <guid-tenant-id>
+  subscriptionId: <guid-subscription-id>
+  resourceGroupName: PromitorRG
+metricDefaults:
+  aggregation:
+    interval: 00:05:00
+  scraping:
+    schedule: "* * * * *"
+metrics:
+  - name: demo_queue_size
+    description: "Amount of active messages of the 'demo_queue' queue"
+    resourceType: ServiceBusQueue
+    namespace: <service-bus-namespace>
+    queueName: demo_queue
+    azureMetricConfiguration:
+      metricName: ActiveMessages
+      aggregation:
+        type: Total
+```
+
+With this file created, you should be able to deploy Promitor with `helm install`. You'll need to be in the top level directory of the Promitor repository for this command to run, or you'll need to edit the chart location.
+
+```bash
+helm install ./charts/promitor-agent-scraper \
+  --name promitor-agent-scraper \
+  --values your/path/to/metric-declaration.yaml
+```
+
+## Install Prometheus
+
+**Note: If you're seeing errors installing Prometheus or Grafana from the Helm chart repository, make sure you run `helm repo update` before digging into the errors more. You might have an outdated copy of the chart.**
+
+Running the deployment command from the previous section should give you an output that includes a script similar to this one:
+
+```bash
+cat > promitor-scrape-config.yaml <<EOF
+extraScrapeConfigs: |
+  - job_name: promitor-agent-scraper
+    metrics_path: /metrics
+    static_configs:
+      - targets:
+        - promitor-agent-scraper.default.svc.cluster.local:80
+EOF
+helm install stable/prometheus -f promitor-scrape-config.yaml
+```
+
+(You can see this output again at any time by running `helm status promitor-agent-scraper`.)
+
+Running these commands will create a Prometheus scraping configuration file in your current directory and deploy Prometheus to your cluster with that scraping configuration in addition to the default.
+
+## Add load to the queue
+
+Now we'll add load to our Service Bus queue so there are meaningful metrics for Promitor to pick up. The easiest way to do this is via the [Service Bus Explorer](https://github.com/paolosalvatori/ServiceBusExplorer/releases). Download and unzip the latest release and run the executable inside.
```

Comment:
```suggestion
Now we'll add load to our Service Bus queue so there are meaningful metrics for Promitor to pick up.

The easiest way to do this is via the [Service Bus Explorer](https://github.com/paolosalvatori/ServiceBusExplorer/releases) where you can download, unzip and run the latest release as an executable.

Alternatively you can install it via Chocolatey:
<code sample but suggestions doesn't like this>
choco install servicebusexplorer
<code sample but suggestions doesn't like this>
```

Reply:
Moved Service Bus Explorer install to the prereq section instead